stan/math/prim/mat/fun/matrix_exp_multiply.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef STAN_MATH_PRIM_MAT_FUN_MATRIX_EXP_MULTIPLY_HPP
+#define STAN_MATH_PRIM_MAT_FUN_MATRIX_EXP_MULTIPLY_HPP
+
+#include <stan/math/rev/mat/fun/matrix_exp_action_handler.hpp>
+
+namespace stan {
+namespace math {
+
+/**
+ * Return product of exp(A) and B, where A is a NxN double matrix,
+ * B is a NxCb double matrix, and t is a double
+ * @tparam N Rows and cols matrix A, also rows of matrix B
+ * @tparam Cb Columns matrix B
+ * @param[in] A Matrix
+ * @param[in] B Matrix
+ * @return exponential of A multiplies B
+ */
+template <int N, int Cb>
+inline Eigen::Matrix<double, N, Cb> matrix_exp_multiply(
+    const Eigen::Matrix<double, N, N>& A,
```

Comment:
Is there any situation where `N != -1`?  If not, then remove the template argument here and make this `const Eigen::MatrixXd&` and the next `Eigen::Matrix<-1, Cb>&`.

Reply:
fixed
```c++
template <int Cb>
inline Eigen::Matrix<double, -1, Cb> matrix_exp_multiply(
    const Eigen::MatrixXd& A, const Eigen::Matrix<double, -1, Cb>& B) {
```